openedx/features/assessment/helpers.py

```diff
@@ -54,21 +55,32 @@ def can_auto_score_ora(enrollment, course, block, index_chapter):
     delta_days = today - enrollment.created.date()
     response_submission_delta = today - response_submission.created_at.date()
 
-    # check if this chapter is 2 weeks older or not.
     module_access_days = delta_days.days - (index_chapter * 7)
     waiting_for_others_submission_exists = AssessmentWorkflow.objects.filter(
         status=ASSESSMENT_WORKFLOW_WAITING_STATUS,
         course_id=course.id,
         item_id=block,
         submission_uuid=response_submission.uuid
     ).exists()
+    days_to_wait_auto_assessment = get_ora_days_to_wait_from_site_configurations()
     return (
-        module_access_days >= DAYS_TO_WAIT_AUTO_ASSESSMENT and
-        response_submission_delta.days >= DAYS_TO_WAIT_AUTO_ASSESSMENT and
+        module_access_days >= days_to_wait_auto_assessment and
+        response_submission_delta.days >= days_to_wait_auto_assessment and
         waiting_for_others_submission_exists
     )
 
 
+def get_ora_days_to_wait_from_site_configurations():
```

Comment:
do we really need this method? As it is only calling another method with default value? 
How about directly calling it where it is required just like it is done in edx code for different site configurations
```
    days_to_wait_auto_assessment = configuration_helpers.get_value('DAYS_TO_WAIT_AUTO_ASSESSMENT',
                                                                 DAYS_TO_WAIT_AUTO_ASSESSMENT
                                                          )
```

Reply:
This function is written separately as per the clean code suggestion, the original method is already very complex. Secondly, the unit test for these helpers methods are in ready for dev at the same time, so separate method helps @monofal not to worry about too many changes, in same function for which he is writing unit tests.

This logic/function might change in the next story.